ecs-cluster/alerts/remedies/12005_TARGET_OBJECTSTORE_FULL_issue.yaml

```
@@ -0,0 +1,6 @@
+- symptomid: OBJST-12005
+  description: Unable to write object to replication destination object store, due to destination object store is full.
+  remedies:
+    -  Add more space to destination object store
+    -  Clean up some of the data in that objectstore to free up space for replication to continue
```

```suggestion
    - Add more space to the destination object store
    - Clean up some of the data in that object store to free up space for replication to continue
```
object store is always 2 words when used in sentences